src/markdown/projects.md

```diff
@@ -1,18 +1,21 @@
-Code for Denver partnered with organizations on these projects last year:
-- [Colorado Anti Violence Program](http://coavp.org/)
-Responsive tool that streamlines internal reporting and external intake forms.
+Code for Denver partnered with organizations on these projects last year:</br>
 
-- [Rocky Mountain MicroFinance Institute](http://www.rmmfi.org/)
-Internal workflow for volunteer relationship management.
-
-- [Fresh Food Connect](http://www.freshfoodconnect.org/#/)
-Transactional web pages for connecting urban gardeners, delivery agents (cyclists), and organization for re-distribution of food.
-
-- [Rocky Mountain Farmers Union](https://feed.rmfu.org)
-An interactive series of web pages, mobile ready/responsive that featured communication through a “flow” of information most important to members.
-
-- [City of Denver Energy Efficiency](?)
-A map of Energy Star rated buildings in Denver.
+- <b>Owlet</b></br>
```

Comment:
I'm not sure if these will render properly on our website. Can we change `<b>` tags to `**` (double asterisks on each side of the word/phrase), and convert `<i>` tags to single underscores on both sides of the word (`_`)?